easypost/requestor.py

```diff
@@ -0,0 +1,193 @@
+import datetime
+import json
+import platform
+import ssl
+import time
+from urllib.parse import urlencode
+
+from easypost.easypost_object import EasyPostObject
+from easypost.error import Error
+
+
+class Requestor(object):
```

Comment:
(and elsewhere) Python3 classes don't benefit from inheriting object.

Reply:
Good call